packages/cli/commands/project/migrateApp.js

```diff
@@ -80,62 +82,65 @@ exports.handler = async options => {
           migrateApp: true,
         });
 
-  const publicApps = await fetchPublicAppOptions(accountId, accountName);
-  const selectedApp = publicApps.find(a => a.id === appId);
-  const appName = selectedApp ? selectedApp.name : 'app';
-  if (!selectedApp) {
-    logger.error(i18n(`${i18nKey}.errors.invalidAppId`, { appId }));
-    process.exit(EXIT_CODES.ERROR);
-  }
-  if (selectedApp.listingInfo) {
-    logger.error(i18n(`${i18nKey}.errors.invalidMarketplaceApp`, { appId }));
-    process.exit(EXIT_CODES.ERROR);
-  }
-
-  const { name, location } = await createProjectPrompt('', options, true);
+  try {
+    const selectedApp = await fetchPublicAppMetadata(appId, accountId);
+    const appName = selectedApp ? selectedApp.name : 'app';
```

Comment:
Is it possible for apps to not have a name?

Reply:
I think apps must have names. Let me check this again, because I think I did this to avoid a TypeScript error.

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1111/commits/47aefe6e63ac137ba4ec03a9acb7668c509a27b6